Modules/Office365DSC/Examples/MSFT_TeamsMemberSettings/MSFT_TeamsMemberSettings.ps1

```diff
@@ -0,0 +1,36 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration TeamsMemberSettingsConfig
+{
+    Import-DSCResource -ModuleName Office365DSC
+    $credsGlobalAdmin = Get-Credential -UserName "TenantAdmin@O365DSC1.onmicrosoft.com" -Message "Global Admin"
+    Node localhost
+    {
+        TeamsMemberSettings MyTeamMemberSettings
+        {
+            TeamName                          = "Sample3"
+            AllowCreateUpdateChannels         = $false
+            AllowDeleteChannels               = $false
+            AllowAddRemoveApps                = $false
+            AllowCreateUpdateRemoveTabs       = $false
+            AllowCreateUpdateRemoveConnectors = $false
+            Ensure                            = "Present"
+            GlobalAdminAccount                = $credsGlobalAdmin
+        }
+    }
+}
+
+$configData = @{
+    AllNodes = @(
+        @{
+            NodeName                    = "localhost"
+            PSDscAllowPlainTextPassword = $true;
```

Comment:
Remove ';' at the end, not needed. I will clean the other examples at some point

Reply:
Done